fireblocks_sdk/sdk.py

```diff
@@ -112,6 +113,41 @@ def link_token(self, asset_id: str):
     def unlink_token(self, asset_id: str):
         return self._delete_request(f"/v1/tokenization/tokens/{asset_id}")
 
+    """Get all staking chains."""
+    def get_staking_chains(self):
+        return self._get_request(f"/v1/staking/chains")
+
+    """Get chain info."""
+    def get_staking_chain_info(self, chain_descriptor: str):
+        return self._get_request(f"/v1/staking/chains/{chain_descriptor}/chainInfo")
+
+    """Get staking positions summary."""
+    def get_staking_positions_summary(self, by_vault: bool = None):
+        return self._get_request(f"/v1/staking/positions/summary",
+                                 query_params={"byVault": "true" if by_vault else "false"} if by_vault else None)
```

Comment:
It'll always be `"true"` if `by_vault` is truthy
```suggestion
                                 query_params={"byVault": "true"} if by_vault else None)
```